Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -255,9 +255,26 @@ def test_section_tags_added(self, dummy_pack_metadata, dummy_pack):
                 "Test Pack Name"
             ]
         }
-        tags = dummy_pack._collect_pack_tags(dummy_pack_metadata, section_tags, [])
+        tags = dummy_pack._collect_pack_tags(dummy_pack_metadata, section_tags, [], 'xsoar')
         assert 'Featured' in tags
 
+    @pytest.mark.parametrize('pack_metadata,marketplace,expected_result',
+                             [({'tags': ['tag1', 'marketplacev2:tag2']}, 'xsoar', {'tag1'}),
+                              ({'tags': ['tag1', 'marketplacev2:tag2']}, 'marketplacev2', {'tag1', 'tag2'}),
+                              ({'tags': ['marketplacev2:tag2']}, 'xsoar', set()),
+                              ({'tags': ['tag1', 'marketplacev2,xsoar:tag2']}, 'xsoar', {'tag1', 'tag2'})])
```

Comment:
Add a test for non-existent marketplace.

Reply:
Actually if we got to this point, the marketplaces are always valid (There will be a validation in the demisto-sdk for it)